src/libcommonserver/log/log.cpp

```diff
@@ -42,10 +42,14 @@ Log::~Log() {
 std::shared_ptr<Log> Log::instance(const log4cplus::tstring &filePath) {
     if (_instance == nullptr) {
         if (filePath.empty()) {
-            throw std::runtime_error("Log must be initialized!");
-        } else {
+            return nullptr;
```

Comment:
Done

Reply:
I removed the assert because it's a normal case:
```
IoHelper::logDirectoryPath(SyncPath &directoryPath, IoError &ioError) noexcept {
    if (Log::instance())...
```